packages/web/src/components/gcds-pagination/gcds-pagination.css

```diff
@@ -76,145 +60,168 @@
         &.gcds-pagination-list-mobile-ellipses {
           display: none;
         }
+      }
+    }
 
-        /* Mobile styling */
-        @media screen and (max-width: 55.625em) {
-          a {
-            border: var(--gcds-pagination-border-width) solid currentColor;
-
-            &:focus {
-              border-color: var(--gcds-pagination-focus-background);
-            }
-          }
+    /* GOOD TO GO */
```

Comment:
Idk if you want to keep this GOOD TO GO comment :)

Reply:
Lol oops